.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@
 
 ## Does this affect other platforms - Amp, Apps, etc?
 
+## Does this affect GLabs Paid Content Pages?
```

Comment:
Good. Is it possible to hint at the 'should have support for Paid Content' angle too?